src/tsickle_compiler_host.ts

```diff
@@ -0,0 +1,259 @@
+import * as path from 'path';
+import {SourceMapConsumer, SourceMapGenerator} from 'source-map';
+import * as ts from 'typescript';
+
+import {convertDecorators} from './decorator-annotator';
+import {processES5} from './es5processor';
+import {ModulesManifest} from './modules_manifest';
+import {annotate} from './tsickle';
+
+export enum Pass {
```

Comment:
Please add docs on the processing model implied by this, it's not immediately clear what a pass is, when it happens, etc.